resources/potw.html

```diff
@@ -10,6 +10,8 @@ <h1>POTW (Problems Of The Week)</h1>
     </div>
 </div>
 
+<div class="row">
+<div class="col-lg-8 col-lg-offset-2">
```

Comment:
```suggestion
	<div class="col-lg-8 col-lg-offset-2">
```
Just make sure to take care of indentations for the page in future.